src/setup-actions.ts

```diff
@@ -0,0 +1,21 @@
+import * as core from "@actions/core";
+import * as installer from "./installer";
+
+async function run() {
+  try {
+    //
+    // Version is optional.  If supplied, install / use from the tool cache
+    // If not supplied then task is still used to setup proxy, auth, etc...
+    //
+    let version = core.getInput("deno-version");
+    core.debug(`Installing deno at version ${version}`);
+    if (version) {
```

Comment:
What happens if there is no version specified here?

Is it better to throw an exception?

Reply:
There should always a version specified because the action.yml has a `default` value for deno-version. Just in case I will throw when no version is specified.